agent/util-scripts/test-bin/rpmdev-vercmp

```diff
@@ -3,18 +3,27 @@
 args="${@}"
 echo "${0} ${args}" >> ${_testlog}
 
+# We are mocking the behavior of rpmdev-vercmp where the behavior of this
+# mock is directed by the 2nd arguments name.  This allows specific test
+# cases to request a certain behavior from rpmdev-vercmp.
+
 if [[ "${2}" == "not-greater-equal-1.0.0" ]]; then
+    echo "${1} <= ${2}"
```

Comment:
If the intention is that the comparison is "not greater than nor equal", then shouldn't the echo'd string be `<` rather than `<=`?

Reply:
Per the comment in `agent/util-scripts/require-rpm`, the return value of `12` is `<`, not `<=` (i.e., if the comparison of `==` were true, the return value would be `0`).